Dockerfile

```diff
@@ -1,53 +1,30 @@
-FROM mambaorg/micromamba:git-0f27156 AS build-env
-
-ENV PYTHONDONTWRITEBYTECODE=1
+FROM ghcr.io/prefix-dev/pixi:0.26.1 AS build
 USER root
 
 # make sure the install below is not cached by docker
-ADD https://loripsum.net/api /opt/docker/etc/gibberish-to-bust-docker-image-cache
```

Comment:
This line prevents docker from caching the steps below during rebuilds. Is there a reason we don't still need it?

Reply:
The lockfile should be everything we need to break the cache, no?